yandex_contest/monitoring.py

```diff
@@ -0,0 +1,19 @@
+def main():
+    rows_number = int(input())
+    colums_number = int(input())
+    if rows_number == 0 or colums_number == 0:
+        return None
```

Comment:
[tiny] this function doesn't return any value, so you can just do
```suggestion
        return
```